{"uid":"9d8b5df4bccee70b","name":"test_view_account_incorrect_authorization","fullName":"test.Cuentas.test_view_account#test_view_account_incorrect_authorization","historyId":"08a118df802ee4b204661dfeb3a1fe83","time":{"start":1726762249061,"stop":1726762249061,"duration":0},"status":"broken","statusMessage":"TypeError: list indices must be integers or slices, not str","statusTrace":"setup_account_view_account = ({'Espo-Authorization': 'YWRtaW46ODAwNDkwNTM=', 'X-Api-Key': '4c29fe9482f3bccb7ac0a7d04015a2e7'}, [{'assignedUserId': None, 'assignedUserName': None, 'billingAddressCity': None, 'billingAddressCountry': None, ...}])\nget_headers = <function get_headers.<locals>._get_headers at 0x7f8c24b5be20>\n\n    @pytest.mark.regression\n    @pytest.mark.functional\n    def test_view_account_incorrect_authorization(setup_account_view_account, get_headers):\n        headers, account = setup_account_view_account\n        headersInvalid = Auth().get_invalid_user_headers(get_headers)\n>       response = EspocrmRequest().get(EndpointAccount.view(account['id']), headersInvalid)\nE       TypeError: list indices must be integers or slices, not str\n\ntest/Cuentas/test_view_account.py:43: TypeError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"get_headers","time":{"start":1726762227223,"stop":1726762227224,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"setup_account_view_account","time":{"start":1726762248893,"stop":1726762249060,"duration":167},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"afterStages":[{"name":"setup_account_view_account::0","time":{"start":1726762249063,"stop":1726762249063,"duration":0},"status":"broken","statusMessage":"TypeError: list indices must be integers or slices, not str\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.10.15/x64/lib/python3.10/site-packages/allure_commons/_allure.py\", line 221, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"/opt/hostedtoolcache/Python/3.10.15/x64/lib/python3.10/site-packages/_pytest/fixtures.py\", line 907, in _teardown_yield_fixture\n    next(it)\n  File \"/home/runner/work/espo-automation-final/espo-automation-final/test/Cuentas/conftest.py\", line 75, in setup_account_view_account\n    AccountCall().delete(headers, account['id'])\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true}],"labels":[{"name":"tag","value":"functional"},{"name":"tag","value":"regression"},{"name":"parentSuite","value":"test.Cuentas"},{"name":"suite","value":"test_view_account"},{"name":"host","value":"fv-az1116-0"},{"name":"thread","value":"2157-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"test.Cuentas.test_view_account"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":["functional","regression"]},"source":"9d8b5df4bccee70b.json","parameterValues":[]}